package.json

```diff
@@ -50,7 +50,7 @@
     "test:performance:server": "serve test/performance -p 5001",
     "test:argos": "node ./scripts/pushArgos.mjs",
     "typescript": "lerna run --no-bail --parallel typescript",
-    "typescript:ci": "lerna run --concurrency 7 --no-bail --no-sort typescript",
+    "typescript:ci": "lerna run --concurrency 6 --no-bail --no-sort typescript",
```

Comment:
Needed to reduce concurrency as a quick fix to running out of memory issues on CI.
FYI: on `next` branch it's set at `4` because pickers have ditched `skipLibCheck` 🙈